app/views/case_workers/claims/_claims_list.html.haml

```diff
@@ -45,6 +45,8 @@
                         'aria-label': "View #{claim.state.humanize} Claim, Case number: #{claim.case_number}"}
             %span.unique-id-small
               = claim.unique_id
+              %br/
+              = claim.type == 'Claim::AdvocateClaim' ? 'AGFS' : 'LGFS'
             - if claim.disk_evidence == true
```

Comment:
This is likely not the right / most reliable way to do this... but to show the requirement it works

Reply:
try 
```ruby
= claim.agfs?
```